compiler-core/src/type_/pipe.rs

```diff
@@ -18,17 +18,24 @@ impl<'a, 'b, 'c> PipeTyper<'a, 'b, 'c> {
         expressions: Vec1<UntypedExpr>,
     ) -> Result<TypedExpr, Error> {
         let size = expressions.len();
-        let mut expressions = expressions.into_iter();
-        let first = expr_typer.infer(
+        let end = expressions
+            .iter()
+            .last()
```

Comment:
Tiny tiny nit but I think we can call the `.last()` methon on a slice to avoid iterating over the prior expressions.

https://doc.rust-lang.org/std/primitive.slice.html#method.last

Reply:
Updated!